### PR TITLE
Add GEM elements in HDQM

### PR DESCRIPTION
**Pull request by SeungJunLee0**
i add the GEM elements in HDQM.

if it cause problem, please send the mail to me.

seungjun.lee@cern.ch
seungjunlee15@gmail.com

### CODE REVIEW

File: backend/extractor/hdqmextract.py [hunk 1]
```diff
@@ -24,7 +24,8 @@
 from helpers import batch_iterable, exec_transaction, get_all_me_names
 
 CFGFILES = 'cfg/*/*.ini'
-ROOTFILES = '/eos/cms/store/group/comm_dqm/DQMGUI_data/*/*/*/DQM*.root'
+#ROOTFILES = '/eos/cms/store/group/comm_dqm/DQMGUI_data/*/*/*/DQM*.root'
+ROOTFILES = '/afs/cern.ch/user/s/seungjun/public/koco/*/*/*/DQM*.root'
```
Comment (jfernan2) [inline backend/extractor/hdqmextract.py]: I believe this line may cause trouble for the rest of subsystems. We may copy the GEM root files to the central eos DQM folder instead.
What do you think @andrius-k  ?

Comment (SeungJunLee0) [reply backend/extractor/hdqmextract.py]: i am sorry that is my mistake..
i fix it

**Comment by jfernan2** [comment]
@SeungJunLee0  is the PR final?
Have you tested it somehow?

**Comment by SeungJunLee0** [comment]
@jfernan2 
yep. i ran the HDQM locally.  
what is the PR??
스크린샷 2021-02-16 오전 3 57 12

**Comment by jfernan2** [comment]
PR = Pull Request :-)
I saw you made a commit just an hour ago, I wonder if the changes are kind of final or if you expect to commit more changes.

**Comment by SeungJunLee0** [comment]
@jfernan2 
Thanks. but it isnt final PR. because MWGR doesnt finish. when new gem elements add in new MWGR or run data, i will implement them.

**Comment by jfernan2** [comment]
OK, so then we wait for merging until you commit all the changes.
Thanks

**Pull request merged**